main.c

```diff
@@ -230,5 +298,99 @@ int main(int argc, char **argv)
     // traverse tree, get encoding
     get_encoding(root, bit_array, 0, encoding);
 
-    test_encoding_map(encoding); // uncomment to test encoding map
+    // test_encoding_map(encoding); // uncomment to test encoding map
+
+    // create new output file here
+    FILE* out_fp = fopen("huffman_encoding_out.txt", "w+");
```

Comment:
Should we just make the output stdout?

Reply:
i think writing the encoding to stdout will
1. interfere with tests which print to stdout
2. makes testing the encoding more difficult, since you can't automatically read the encoding in from a file

but i'm not really opposed to it